src/app/components/LimitIE9Comment/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+/*
+ * These, when used together, stop anything within them from loading on IE9 and below.
+ */
+
+export const IE9LimitOpen = () => (
```

Comment:
```suggestion
export const ifNotIE9Open = () => (
```

This feels more explicitly in what the component is doing

Reply:
Done :)